jerry-core/include/jerryscript-core.h

```diff
@@ -321,6 +321,16 @@ jerry_size_t jerry_substring_to_utf8_char_buffer (const jerry_value_t value,
                                                   jerry_char_t *buffer_p,
                                                   jerry_size_t buffer_size);
 
+#define DECLARE_C_STRING_FROM_JERRY_STRING(prefix, value)               \
+  jerry_size_t prefix##_size = jerry_get_string_size ((value));         \
+  jerry_char_t prefix##_string[prefix##_size];                          \
+  jerry_string_to_char_buffer ((value), prefix##_string, prefix##_size)
+
+#define DECLARE_C_STRING_FROM_UTF8_JERRY_STRING(prefix, value)               \
+  jerry_size_t prefix##_size = jerry_get_string_size ((value));              \
+  jerry_char_t prefix##_string[prefix##_size];                               \
```

Comment:
Hmm, I'm not sure if it's a good idea to have these.
The stack frames will blow up a lot when the strings are large.
Unless the string size is known to be small, I think it's kind of an anti-pattern, tbh.

Reply:
I have a similar opinion.